AirCasting/Settings/MyAccountViewSingOut.swift

```diff
@@ -0,0 +1,40 @@
+// Created by Lunar on 18/06/2021.
+//
+
+import SwiftUI
+
+struct MyAccountViewSingOut: View {
+    var body: some View {
```

Comment:
here the `body` is less complicated, but IMO would still benefit from making same thing as above (your call)